### ISSUE TITLE
A11y: Cart page - Shopping cart table doesn't have a sufficient semantic html structure

### ISSUE THREAD

**Issue by tieupmedia** [shopware/platform#5223]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.6.7  | *Affected area / extension:* Platform(Default) 

 ----

*_Expected behaviour:_*

The shopping cart table is structured as a correct HTML table. Pay particular attention to set the table headers correctly. The image and the title of the product should be in a “Product” table column, just as it is visually designed. The action for deletion can be in a separate table column with the table header “Actions”. The table header can be made visually invisible to resemble the previous appearance.

---

Ressources: https://www.w3.org/WAI/tutorials/tables/

*_Actual behaviour:_*

On the cart page the items are displayed in a tabluar view. This tabular structuring of the contents is not represented programmatically in the source code. There it is only a collection of `div`. The correct labeling in the source code is also important, as screen reader users are then offered further navigation options. Similar to how you can navigate in an Excel spreadsheet with arrow keys.

---

Violation relating to [EN 301 549](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf#%5B%7B%22num%22%3A120%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C54%2C517%2C0%5D), https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1#info-and-relationships and https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

*_How to reproduce:_*

Go to the default storefront and open the cart page. Inspect the source code.

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39187**



[created from NEXT-39187, comment 505777]

**Comment by timshopware** [comment]
Hey, that Issue is closed is a mistake from our github bot. We are currrently working on this in another ticket which was not synced correctly with Github. We keep you updated as soon this is solved
